backend/apps/notifications/middleware.py

```
@@ -1,64 +1,37 @@
+        if not token:
+            query_string = scope.get('query_string', b'').decode()
+            params = dict(q.split('=') for q in query_string.split('&') if '=' in q)
+            token = params.get('token')
+        
+        # Validate token
+        if token:
+            try:
+                payload = jwt.decode(token, settings.SECRET_KEY, algorithms=['HS256'])
+                user = await database_sync_to_async(User.objects.get)(id=payload['user_id'])
+                scope['user'] = user
+            except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)